core/src/main/java/org/apache/iceberg/ContentFileParser.java

```diff
@@ -84,9 +88,12 @@ public static void toJson(ContentFile<?> contentFile, PartitionSpec spec, JsonGe
     // as it isn't used and BaseFile constructor doesn't support it.
 
     generator.writeNumberField(SPEC_ID, contentFile.specId());
-    generator.writeStringField(CONTENT, contentFile.content().name());
+    // Since 1.11, we serialize content as lowercase kebab-case values like "equality-deletes"
+    generator.writeStringField(
+        CONTENT, contentFile.content().name().toLowerCase(Locale.ROOT).replace('_', '-'));
     generator.writeStringField(FILE_PATH, contentFile.location());
-    generator.writeStringField(FILE_FORMAT, contentFile.format().name());
+    // Since 1.11, we serialize format as lower-case strings like "parquet"
+    generator.writeStringField(FILE_FORMAT, contentFile.format().name().toLowerCase(Locale.ROOT));
```

Comment:
I think we may actually want to use `Locale.ENGLISH` here and a few lines above this